src/ape/api/providers.py

```diff
@@ -136,10 +136,24 @@ def decode(cls, data: dict) -> "ReceiptAPI":
         ...
 
 
+@abstractdataclass
+class BlockGasAPI:
+    raw_data: Dict
+
+    @property
+    @abstractmethod
+    def base_fee(self) -> Optional[int]:
+        ...
+
+    @property
+    @abstractmethod
+    def total_gas_used(self) -> int:
+        ...
+
+
 @abstractdataclass
 class BlockAPI:
-    gas_fee_data: Dict
-    consensus_data: Dict
+    gas_fee_data: BlockGasAPI
```

Comment:
seems like this should just be `gas_data` come to think of it

or maybe just `gas` (`block.gas.total_gas_used` :thinking:)

Reply:
done!